ljclang.lua

```diff
@@ -920,5 +930,10 @@ ffi.metatype(CompileCommand_t, CompileCommand_mt)
 ffi.metatype(CompileCommands_t, CompileCommands_mt)
 ffi.metatype(CompilationDatabase_t, CompilationDatabase_mt)
 
+api.Index_t = Index_t
```

Comment:
(Replying to myself:) OK, probably for quickly testing out functinality, right? In that case I'm OK to keep it as a development tool, but not expose it in the documentation.